src/vivarium_census_prl_synth_pop/data/nicknames_proportion.py

```diff
@@ -1,15 +1,26 @@
+from itertools import chain
+from pathlib import Path
+
 import pandas as pd
 from vivarium import Artifact
 
 from vivarium_census_prl_synth_pop.constants import data_keys, paths
 
 
-def get_nicknames_proportion():
+def get_nicknames_proportion(data_dir: Path) -> float:
     # This function calculates the proportion of names that have potential nicknames. This will return a float that
     # that will be used as a constant to scale the nicknames noise function in the pseudopeople packages.
 
-    art = Artifact(paths.DEFAULT_ARTIFACT)
-    names = art.load(data_keys.SYNTHETIC_DATA.FIRST_NAMES).reset_index()["name"]
+    data_files = []
+    for file in data_dir.glob("*.parquet"):
+        data_files.append(file)
+    names = []
+    for file in data_files:
+        df = pd.read_parquet(file)
+        df = df.loc[df["year"] == 2030, "first_name"]
```

Comment:
Don't worry about changing this, but if we re-run this, we should use the strategy to load only the 2030 year from the file that we are already using to load in our interface functions.